lib/std/pointers.nim

```diff
@@ -0,0 +1,15 @@
+##[
+Convenience procs to deal with pointer-like variables.
+]##
+
+proc toUncheckedArray*[T](a: ptr[T]): ptr UncheckedArray[T] {.inline.} =
```

Comment:
```suggestion
proc toUncheckedArray*[T](a: ptr T): ptr UncheckedArray[T] {.inline.} =
```

In idiomatic Nim we rarely use `ptr[T]` in that form.

Reply:
fixed